x-pack/plugins/security/public/management/api_keys/api_keys_management_app.test.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+jest.mock('./api_keys_grid', () => ({
```

Comment:
__note:__ I couldn't find a better way to test this as `ReactDom.render` is called inside of `mount`. It tests what I wanted it to test though, but if it feels weird to you just tell me!

Reply:
works for me!